src/NHibernate/Impl/SessionIdLoggingContext.cs

```diff
@@ -8,9 +8,15 @@ public class SessionIdLoggingContext : IDisposable
 		private static readonly AsyncLocal<Guid?> _currentSessionId = new AsyncLocal<Guid?>();
 
 		private readonly Guid? _oldSessonId;
+		private static bool _tracking;
```

Comment:
`static` is probably forgotten from previous version, it needs to be removed.

Reply:
[Replaced](https://github.com/nhibernate/nhibernate-core/pull/1394/commits/8ca4ba8e4a32a800bf2a48de9b371387e4f034ae#diff-582ed89a24df0b05c4a1198ffb0f8e1cR11) static with readonly.